go/enclave/db/storage.go

```diff
@@ -210,11 +198,18 @@ func (s *storageImpl) IsBlockAncestor(block *types.Block, maybeAncestor common.L
 }
 
 func (s *storageImpl) HealthCheck() (bool, error) {
-	headRollup, err := s.FetchHeadRollup()
+	_, l2Head, err := s.FetchCurrentHeads()
```

Comment:
Is it worth keeping the FetchHeadRollup method and using this code as its body? Looks like it's used twice and I'm sure it'll be used more in future